bigbone/src/main/kotlin/social/bigbone/api/entity/Application.kt

```diff
@@ -7,5 +7,14 @@ data class Application(
     val name: String = "",
 
     @SerializedName("website")
-    val website: String? = null
+    val website: String? = null,
```

Comment:
Could you add the KDoc here to these class members? I took them from the Mastodon API docs (entities).

Reply:
Of course, added now. :)